imagery/i.ortho.photo/lib/find_init.c

```diff
@@ -8,11 +8,16 @@
 int I_find_initial(char *group)
 {
     char *element;
+    int file_exists;
 
     element = (char *)G_malloc(80 * sizeof(char));
 
-    if (group == NULL || *group == 0)
+    if (group == NULL || *group == 0) {
+        G_free(element);
         return 0;
+    }
     sprintf(element, "group/%s", group);
```

Comment:
For another PR: use snprintf instead of sprintf.

```suggestion
        snprintf(element, ELEMENT_BUFFER_SIZE, "group/%s", group);
```

with #define ELEMENT_BUFFER_SIZE 80 also used for malloc instead of a magic number

Reply:
ok i will create another PR with this now

Reply:
done changes